HaxeManual/08-compiler-features.tex

```diff
@@ -143,7 +143,7 @@ \section{Dead Code Elimination}
 \begin{description}
 	\item[\expr{@:keep}:] If used on a class, the class along with all fields is unaffected by DCE. If used on a field, that field is unaffected by DCE.
 	\item[\expr{@:keepSub}:] If used on a class, it works like \expr{@:keep} on the annotated class as well as all subclasses.
-	\item[\expr{@:keepInit}:] Usually, a class which had all fields removed by DCE (or is empty to begin with) is removed from the output. By using this metadata, empty classes are kept.
+	\item[\expr{@:keepInit}:] Usually, a class which had all fields removed by DCE (or is empty, to begin with) is removed from the output. By using this metadata, empty classes are kept.
```

Comment:
The comma should not be here. `the class is empty to begin with` (https://dictionary.cambridge.org/dictionary/english/to-begin-with)